jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -517,6 +990,17 @@ private static DiagnosticProperties doCheckpointJVM(String diagnosticCommand) {
 		commandTable.put(DIAGNOSTICS_LOAD_JVMTI_AGENT, DiagnosticUtils::loadJVMTIAgent);
 		helpTable.put(DIAGNOSTICS_LOAD_JVMTI_AGENT, DIAGNOSTICS_LOAD_JVMTI_AGENT_HELP);
 
+/*[IF JFR_SUPPORT]*/
+		commandTable.put(DIAGNOSTICS_JFR_START, DiagnosticUtils::doJFR);
+		helpTable.put(DIAGNOSTICS_JFR_START, DIAGNOSTICS_JFR_START_HELP);
```

Comment:
`DIAGNOSTICS_JFR_START_HELP` was defined for JDK 11/17, it won't compile for JDK 21+.